modules/developer-cli-odo-installing-odo-on-vs-code.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/developer_cli_odo/installing-odo.adoc
+
+[id="installing-odo-on-vs-code_{context}"]
+
+= Installing odo on VS Code
+
+The https://marketplace.visualstudio.com/items?itemName=redhat.vscode-openshift-connector[OpenShift VS Code extension] uses both `odo` and the `oc` binary to interact with your {product title} cluster. Install the OpenShift VS Code extension on VS Code.
+
+== Prerequisites
```

Comment:
This needs to be `.Prerequisites`. See the mod docs template file here for details: https://raw.githubusercontent.com/redhat-documentation/modular-docs/master/modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc

Reply:
Okay, would that mean all the other sections, need a `.` instead of a `=` or `==`?

Reply:
".Procedure" and ".Prerequisites" are special headings that are part of the mod docs template for a procedure module. Same for if you were to have  ".Verification steps" or ".Additional resources" in the module.

Now if you're in an assembly, you would use "== Prerequisites" and "== Additional resources".

Otherwise, if you just have any other random heading outside of these ones that are part of the template, then yeah normally you would use the == Heading format.